### DISCUSSION TITLE
Gemmy_TTRPG Repo

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#19] [The Burned Steak]
I guess here is a great example of us starting our work here. :)

Gemmy_TTRPG is now on brat. We ought to now make sure it shows commit messages/has release notes/changelogs etc.

Any suggestions on the best way to implement this that is Josh Friendly? I'm thinking maybe a combination of [Automatically generated release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) and a GitHub action or two to help.

**Comment by scottTomaszewski** [comment]
What needs to happen in the Github Action?

**Reply by sigrunixia** [reply]
Once it's out of Brat, it will need a release action to trigger updates to Obsidian. During then, would also be a good time to auto compile release notes, generate a changeling.md.. I suppose the normal stuff?

**Reply by sigrunixia** [reply]
https://github.com/Obsidian-TTRPG-Community/13th-Age-SRD-Markdown/releases/tag/untagged-6fb5bfe54de6dad69ce4

Probot might be to the rescue :)
